app/code/core/Mage/Core/Model/Resource/File/Storage/File.php

```diff
@@ -214,6 +208,6 @@ public function saveFile($filePath, $content, $overwrite = false)
             Mage::throwException(Mage::helper('core')->__('Unable to save file: %s', $filePath));
         }
 
-        return false;
+        return true;
```

Comment:
so if the file already exists, we now return true instead of false? Sounds wrong to me, is there a special reason why this is needed?